saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/servlet/filter/Saml2WebSsoAuthenticationFilter.java

```diff
@@ -50,13 +53,21 @@ public Saml2WebSsoAuthenticationFilter(RelyingPartyRegistrationRepository relyin
 	public Saml2WebSsoAuthenticationFilter(
 			RelyingPartyRegistrationRepository relyingPartyRegistrationRepository,
 			String filterProcessesUrl) {
+		this(relyingPartyRegistrationRepository, filterProcessesUrl, null);
+	}
+
+	public Saml2WebSsoAuthenticationFilter(
```

Comment:
Reminder to please add JavaDoc, including a `@since` tag.

Reply:
Completed